tests/data/ssl4eo_downstream_landsat/data.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import shutil
+from typing import Union
+
+import numpy as np
+import rasterio
+from rasterio import Affine
+from rasterio.crs import CRS
+
+IMG_DIR = "ssl4eo-l5-l1-conus"
+MASK_DIR = "l5-*-2011"
+MASKS = ["cdl", "nlcd"]
+
+SUBDIRS = [("0000000", "LT05_045030_20110723"), ("0000001", "LT05_040032_20110805")]
+
+NUM_BANDS = 7
+SIZE = 36
+
+np.random.seed(0)
+
+FILENAME_HIERARCHY = Union[dict[str, "FILENAME_HIERARCHY"], list[str]]
+
+filenames: FILENAME_HIERARCHY = {
+    "tm_toa": {
+        "0000002": {
+            "LE07_172034_20010526": ["all_bands.tif"],
```

Comment:
These directory names should be different

Reply:
Not sure what you mean here.

Reply:
TM is Landsat 5, not Landsat 7

Reply:
But those are the 5 different names for the different sensors we have and I want to create fake data for all of them with one script like in the other `ssl4eo/data.py`.